Makefile

```diff
@@ -95,7 +95,10 @@ bundle:
 
 setup: build db_create db_migrate
 
-publish: build
+migrate_if_needed:
+	bash -c 'if docker-compose run ${RAILS_CONTAINER} rake db:migrate:status | grep "^\s*down"; then docker-compose run ${RAILS_CONTAINER} rake db:migrate; fi'
```

Comment:
I recently discovered the `db: abort_if_pending_migrations` task which pretty much does what it says: exits the rake execution with an error if there are pending migrations.

```
You have 10 pending migrations:
  20180129223616 AddGroupToTeamMembers
  20180204235116 AddMoocToCodeSchools
  20180401034441 CreateRoles
  20180408172532 AddRoleIdToAdminUser
  20180416024105 AddEmailToTeamMember
  20180420192231 AddIsPartnerToCodeSchools
  20180422030623 CreateSlackUser
  20180422031009 DropUserSlackId
  20180430161218 AddRepNameToCodeschool
  20180711212702 AddMilitarystatusToUser
Run `rails db:migrate` to update your database then try again.
```

And now that I know about it, of course I want to use it! What do you think about this?

```
migrate_if_needed:
	bash -c 'docker-compose run ${RAILS_CONTAINER} rake db:abort_if_pending_migrations || docker-compose run ${RAILS_CONTAINER} rake db:migrate'
```